.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,11 +6,9 @@ about: Create a report to help us improve
 <!--
 Welcome! Before creating a new issue please search for relevant issues and recreate the issue in a fresh environment:
 
-1. conda create -n ipywidgets-test --override-channels --strict-channel-priority -c conda-forge -c anaconda jupyterlab nodejs=12
+1. conda create -n ipywidgets-test --override-channels --strict-channel-priority -c conda-forge -c anaconda jupyterlab
```

Comment:
Can we list `mamba` everywhere?

```suggestion
1. mamba create -n ipywidgets-test --override-channels --strict-channel-priority -c conda-forge -c anaconda jupyterlab
```

Reply:
It would make sense yes, or at least as an alternative (if that doesn't complicate the instructions)